psi4/driver/procrouting/proc.py

```diff
@@ -3061,6 +3108,271 @@ def run_adc(name, **kwargs):
     return core.adc(ref_wfn)
 
 
+def run_adcc(name, **kwargs):
+    """Prepare and run an ADC calculation in adcc, interpret the result and return
+    as a wavefunction.
+
+    """
+    # TODO Maybe it would improve readability if this function was spilt
+    #      up and the whole thing went to a separate file (like for sapt,
+    #      interface_cfour.py, ...
+
+    try:
+        import adcc
+        from adcc.backends import InvalidReference
+    except ModuleNotFoundError:
+        raise ValidationError("adcc extras qc_module not available. Try installing "
+            "via 'pip install adcc' or 'conda install -c adcc adcc'.")
```

Comment:
Sorry, I tried to review this over a week ago but I just discovered that comments are not visible to you until I `finalize review` - I thought that button was for final acceptance.  Time for ☕️ 

Anyway, this is fantastic work and please don't mistake my inability to use GitHub reviews correctly with a lack of interest.  This will add a number of great methods and you've done a great job with documentation and tests 😄

Reply:
Yes that's currently still needed to pull adcc from conda.

@maxscheurer Correct me if I am wrong.

Reply:
`conda install -c adcc/label/dev_mkl adcc` would be more suitable at the moment I guess...